### PR TITLE
Simply running bundle update and yarn upgrade to fix some security concerns

### PR DESCRIPTION
**Pull request by murny**
Hopefully fixes acorn and minimist versions to make github security alerts go away

#trivial

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -36,6 +36,12 @@ Layout/LineLength:
 Layout/RescueEnsureAlignment:
   Enabled: false
 
+Lint/RaiseException:
```
Comment (murny) [inline .rubocop.yml:39]: New cops. 

https://rubocop.readthedocs.io/en/latest/cops_lint/#lintraiseexception

and

https://rubocop.readthedocs.io/en/latest/cops_lint/#lintstructnewoverride

File: app/models/jupiter_core/solr_services/facet_result.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 class JupiterCore::SolrServices::FacetResult
 
+  # rubocop:disable Lint/StructNewOverride
```
Comment (murny) [inline app/models/jupiter_core/solr_services/facet_result.rb:3]: We overriding `count` here. This is probably fine. Just whitelisting this

**Comment by pgwillia** [comment]
Resolves #1581

File: app/models/jupiter_core/solr_services/facet_result.rb (hunk 2 above)
Comment (pgwillia) [reply app/models/jupiter_core/solr_services/facet_result.rb:3]: I wasn't sure.  Trust your judgement on this.

Comment (murny) [reply app/models/jupiter_core/solr_services/facet_result.rb:3]: Yeah i'll let @mbarnett  chime in here. 

We can either call this `count` something else so we not overriding it...`results_count` or what not. Then we don't need to add this exception. Or just leave it as is. 

Seems like a silly cop? I think the default `count` of a Struct.New is just the number of Struct Members? (or is that `.length`?)

Comment (lagoan) [reply app/models/jupiter_core/solr_services/facet_result.rb:3]: From https://rubocop.readthedocs.io/en/latest/cops_lint/#lintstructnewoverride

count in Struct new overrides [Enumerable#count](https://ruby-doc.org/core-2.7.1/Enumerable.html#method-i-count)

I would say it is best call it something else as you suggest @murny

Comment (mbarnett) [reply app/models/jupiter_core/solr_services/facet_result.rb:3]: Yeah, just to clarify: we're intentionally overriding Enumerable#count here. This lets the FacetResult play nicely with things that build on the Enumerable machinery (pagination, etc) without having to expose the underlying collection directly.

As for why Rubocop objects to basic encapsulation, ¯\_(ツ)_/¯

**Pull request merged**